src/control/server/README.md

```diff
@@ -1,5 +1,11 @@
 # DAOS Server (control-plane)
 
+## Workflow
+
+Control plane server (`daos_server`) instances will open and listen on gRPC channel for requests from control plane client applications. Administrators can perform provisioning operations on network and storage hardware through control plane [`dmg`](../dmg) management tool and when ready to start data plane, calling `dmg storage format` will enable the server to be formatted (superblock written) and start the data plane.
```

Comment:
Basic content looks good. A few minor suggestions for clarity:
"will open and listen on gRPC channel for requests" -> "will open a gRPC channel to listen for requests"
"through control plane (dmg) management tool" -> "through the control plane (dmg) management tool"
"and when ready to start data plane, ..." -> New sentence: "Calling `dmg storage format` formats the server, writes the superblock, and starts the data plane."